### ISSUE TITLE
NTR: low PBMC CD18 surface expression

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#4578]
Defining feature of 
https://www.ncbi.nlm.nih.gov/pubmed/12488604/

**Comment by pnrobinson** [comment]
also here for annotations https://www.ncbi.nlm.nih.gov/pubmed/26497373

**Comment by pnrobinson** [comment]
This term can go here:https://hpo.jax.org/app/browse/term/HP:0031553
label: Reduced granulocyte CD18 level
Definition: Reduced level of CD18 on the granulocyte surface. This feature can be assessed by flow cytometry.
Comment: The beta2 integrin (CD18) expressed on leukocytes can pair with several alpha integrin subunits, and each alpha-beta pairing can bind a variety of ligands, including the counterreceptors ICAM-1, -2 and -3, fibrinogen, the complement fragment iC3b, and polysaccharides.

xrref-PMID:20142805
